src/status_im/contexts/chat/messenger/composer/events.cljs

```diff
@@ -186,24 +209,31 @@
     :as   cofx} text {:keys [message-id quoted-message chat-id]}]
   (rf/merge
    cofx
-   {:json-rpc/call [{:method      "wakuext_editMessage"
-                     :params      [{:id           message-id
-                                    :text         text
-                                    :content-type (if (emoji-only-content?
-                                                       {:text        text
-                                                        :response-to quoted-message})
-                                                    constants/content-type-emoji
-                                                    constants/content-type-text)
-                                    :linkPreviews (map #(-> %
-                                                            (select-keys [:url :title :description
-                                                                          :thumbnail])
-                                                            data-store-messages/->link-preview-rpc)
-                                                       (get-in db [:chat/link-previews :unfurled]))}]
-                     :js-response true
-                     :on-error    #(log/error "failed to edit message " %)
-                     :on-success  (fn [result]
-                                    (rf/dispatch [:sanitize-messages-and-process-response
-                                                  result]))}]}
+   {:json-rpc/call
+    [{:method      "wakuext_editMessage"
+      :params      [{:id                 message-id
+                     :text               text
+                     :content-type       (if (emoji-only-content?
+                                              {:text        text
+                                               :response-to quoted-message})
+                                           constants/content-type-emoji
+                                           constants/content-type-text)
+                     :linkPreviews       (->> (get-in db [:chat/link-previews :unfurled])
```

Comment:
This section grew a little bit more, and it could be extracted to a separate function outside the already big event handler. It only depends on the value of `[:chat/link-previews :unfurled]`, so it's easy to extract.